blogs/whats_new_1.5/index.md

```diff
@@ -0,0 +1,178 @@
+_<div style="text-align: center">Figure 7: Single graph focus</div>_
+
+### Traffic flows tab
+
+These are the new labels in the flow data.
+
+1. Differentiated Services Code Point (DSCP)<br>
+This is a 6-bit value in the IP packet header that indicates the priority of a packet to provide quality of service (QoS), particularly for time-sensitive data such as voice and video.  The value "Standard" translates to 0 or best effort.  In other words, the traffic is not getting any special treatment.
+
+    - Column: DSCP
+    - Label: Dscp
+
+![flow table - DSCP](images/flow_table-dscp.png)
+_<div style="text-align: center">Figure 8: DSCP</div>_
+
+2. Availability Zones<br>
+A region defines a geographical area and consists of one or more availability zones.  For example, if a region is named us-west-1, then the zones might be us-west-1a, us-west-1b, and us-west-1c, where each zone might have one or more physical data centers.
+
+    - Columns: Source Zone, Destination Zone
+    - Labels: SrcK8S_Zone, DstK8S_Zone
+
+![flow table - Zone](images/flow_table-zone.png)
+_<div style="text-align: center">Figure 9: Availability Zone</div>_
+
+3. Cluster ID<br>
+The cluster ID is the same value shown in the **Home > Overview**, Details section.
+
+    - Column: Cluster
+    - Label: K8S_ClusterName
+
+![flow table - Cluster ID](images/flow_table-cluster_id.png)
+_<div style="text-align: center">Figure 10: Cluster ID</div>_
+
+#### Manage columns selection
+
+Like the **Manage panels** dialog, the **Manage columns** dialog, found under "Show advanced options", alos provides a simple filter (Figure 11).  Click one or more buttons to filter on the selection.
+
+![Manage Columns](images/manage_columns.png)
+_<div style="text-align: center">Figure 11: Manage columns</div>_
+
+### Topology tab
+
+Topology also supports the same data in the **Overview** graphs for its edge labels, such as P90 (90th percentile) and P99 (99th percentile)
+
+![Topology - Display options](images/topology-display_options.png)
+_<div style="text-align: center">Figure 12: Topology changes</div>_
+
+### Filter
+
+At the top of the screen is the filter used by all three tabs.  A dropdown button was added to show recently-used entries and to do auto-completion (Figure 13).
+
+![filter](images/filter.png)
+_<div style="text-align: center">Figure 13: Filter</div>_
+
+## FlowMetrics API
+
+The FlowMetrics API allows you take any combination of [flow data labels](https://docs.openshift.com/container-platform/4.14/network_observability/json-flows-format-reference.html) and turn it into a Prometheus metric.  In other words, you can create your own custom metric and then even create alerts and external notifications based on them.  This is a development preview feature.  Please be aware that generating too many metrics or not understanding how performance is impacted by querying these metrics can result in over utilization of resources and storage and cause instability.
```

Comment:
This is the right place to give more context on the needs here. 

A use case is where customers doesn't want to install Loki but need to keep an eye on specific metrics.